### ISSUE TITLE
Confusing ERROR: "path _ for package _ no longer exists"

### ISSUE THREAD

**Issue by haberdashPI** [JuliaLang/Pkg.jl#821]
In a package I'm working on, I get the error:

```
ERROR: path /Users/davidlittle/.julia/packages/AuditoryBistabilityLE/Y5EpE for package AuditoryBistabilityLE no longer exists. Remove the package or `develop` it at a new path
```

Yet, I have already removed this package from the current environment (which is the default 0.7 environment).

In trying to troubleshoot what exactly is going on here I ran the following to figure out where the package is being referenced.

```
cd ~/.julia
rg --no-ignore Y5EpE
```

But ripgrep returned no results. Any hints anyone might have as to where I should be looking for the references to this previously removed package? 

Edit: I've also run this search in the current directory where julia was running when I got the error, and used `fd` to search for possible file names in these locations to no avail.

**Comment by fredrikekre** [comment]
It has to be referenced in the `Manifest.toml` file, otherwise where would we read the path from? Can you try
```
pkg> rm --manifest AuditoryBistabilityLE
```
?

**Comment by haberdashPI** [comment]
I resolved my problem at the time by adding the appropriate repository and then removing it again, so I cannot confirm if your solution worked. However, at the time I did check the Manifest file for this package and did not see it (I searched for `AuditoryBistabilityLE` and for `Y5EpE`) so I don't *believe* this would have solved the problem.

I will try to keep an eye out for this problem again and try your solution and see if it works. It is highly possible I am just doing something wrong with the new `Pkg`; I don't understand it yet.
